code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -35,50 +35,39 @@
 	var/datum/action/cooldown/domain/domain
 	///The Spell that the rat uses to recruit/convert more rats.
 	var/datum/action/cooldown/riot/riot
+	///Should we request a mind immediately upon spawning?
+	var/poll_ghosts = FALSE
 
 /mob/living/simple_animal/hostile/regalrat/Initialize(mapload)
 	. = ..()
+	ADD_TRAIT(src, TRAIT_VENTCRAWLER_ALWAYS, INNATE_TRAIT)
+	AddElement(/datum/element/waddling)
+	AddComponent(\
+		/datum/component/ghost_direct_control,\
```

Comment:
Cargorilla should use this too, pretty much the same as the Rat

Reply:
Noticed that myself, already done in incoming commit 👍